python/ray/serve/tests/test_api.py

```diff
@@ -412,6 +412,59 @@ def g():
     assert requests.get("http://127.0.0.1:8000/app_g").text == "got g"
 
 
+@pytest.mark.asyncio
+async def test_delete_while_initializing(serve_instance):
+    @ray.remote
+    class EventHolder:
```

Comment:
Nit: We can probably reuse `SignalActor` which is already imported in this file?